### PR TITLE
feat(cirrus): Test targeting context

### LINKED ISSUE #10147: Test for targeting context
Monitor wants to run experiment and target the users based on `language` i.e. English and based on `country` i.e. US, add some tests on Cirrus so we can test is supports targeting

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-4288)

### PR DESCRIPTION
**Pull request by yashikakhurana**
Because

- Monitor wants to start actual experiments and they want to target by language as `English` and country as `US`

This commit

- Add tests for targeting context so that we should be able to use `language` and `region`
- Update readme 

Fixes #10147

### CODE REVIEW

File: cirrus/server/tests/test_feature_manifest.py [hunk 1]
```diff
@@ -181,6 +181,111 @@ def test_compute_feature_configurations_targeting_doesnt_match(fml_setup):
...
+        ),
+        (
+            '(locale in ["en", "fa"]) && (region in ["CA", "US"])',
+            {
+                "clientId": "test",
+                "requestContext": {"locale": "en", "region": "US"},
+            },
+        ),
+    ],
+)
```
Comment (yashikakhurana) [inline cirrus/server/tests/test_feature_manifest.py:209]: @jeddai @jhugman I am testing targeting as it used SDK, I was hoping `locale` and `region` should work, but for some reason `region` is not working, although `locale` is working fine

**Review by yashikakhurana — COMMENTED** [review]
Looking for feedback from @jhugman @jeddai

**Comment by brennie** [comment]
Your commit message references "locale" but the key being tested is "language".

**Comment by yashikakhurana** [comment]
> Your commit message references "locale" but the key being tested is "language".

thanks, modified now

File: cirrus/README.md [hunk 2]
```diff
@@ -129,7 +129,40 @@ Example input:
...
+Example input:
+```json
+{
+  "client_id": "4a1d71ab-29a2-4c5f-9e1d-9d9df2e6e449",
+  "context": {
+    "language": "en",
+    "region": "US"
+  }
+}
+```
```
Comment (eliserichards) [inline cirrus/README.md:165]: Great examples 👏

**Review by eliserichards — APPROVED** [review]
Awesome! 🚀

**Pull request merged**